test/browsercontext-device.spec.ts

```diff
@@ -58,4 +59,76 @@ describe('device', (suite, { browserName }) => {
     expect(await element.textContent()).toBe('clicked');
     await context.close();
   });
+
+  it('should scroll twice when emulated',(test, {browserName}) => {
+    test.skip(browserName === 'webkit');
+  }, async ({server, contextFactory, playwright, contextOptions}) => {
+    const device = playwright.devices['iPhone 6'];
+    const context = await contextFactory({
+      ...contextOptions,
+      ...device,
+      viewport: {
```

Comment:
Why both device and viewport? Here and everywhere.

Reply:
Video needed even viewport demensions for when I was debugging. Removed.